src/jsMain/js/simulator/windows/ShaderEditorWindow/ShaderEditorWindow.jsx

```diff
@@ -89,6 +97,21 @@ const ShowEditorWindow = (props) => {
     console.log(`previewShow!`);
   };
 
+  const handleTabChange = (event, newValue) => {
+    let src = openShaders[newValue].src;
+    console.log('src', src)
```

Comment:
I think we could remove this `console.log` if it is just being used for debugging?